packages/cashier/src/Sass/cashier.scss

```diff
@@ -1122,6 +1122,79 @@
     }
 }
 
+.cashier-recent-transaction-wrapper {
+    max-width: 256px;
+    @include mobile {
+        max-width: 328px;
```

Comment:
Please use `rem` instead of `px` since `px` is not scalable.

Reply:
Sure